modules/admin_manual/pages/oidc/index.adoc

```diff
@@ -0,0 +1,162 @@
+= What is OpenID Connect (OIDC)
+:toc:
+:panva-node-oidc-provider-url: https://github.com/panva/node-oidc-provider
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+
+According to openid.net/connect:
+
+"OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
+OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
+-- https://openid.net/connect/, "What is OpenID Connect?"
```

Comment:
why not using a macro reference here like we do in all urls ?
`:what-is-open-id-connect-url: https://openid.net/connect/`

Reply:
Good point. Didn't think of that.